ports/vcpkg-make/vcpkg_make_configure.cmake

```
@@ -0,0 +1,136 @@
+include_guard(GLOBAL)
+include("${CMAKE_CURRENT_LIST_DIR}/vcpkg_make.cmake")
+
+function(vcpkg_make_configure) # Replacement for vcpkg_configure_make
+    cmake_parse_arguments(PARSE_ARGV 0 arg
+        "AUTOCONFIG;COPY_SOURCE;DISABLE_MSVC_WRAPPERS;DISABLE_CPPFLAGS;DISABLE_DEFAULT_OPTIONS;DISABLE_MSVC_FLAG_ESCAPING"
+        "SOURCE_PATH"
+        "OPTIONS;OPTIONS_DEBUG;OPTIONS_RELEASE;PRE_CONFIGURE_CMAKE_COMMANDS;LANGUAGES"
+    )
+
+    z_vcpkg_unparsed_args(FATAL_ERROR)
+
+    # Can be set in the triplet to append options for configure
+    if(DEFINED VCPKG_MAKE_CONFIGURE_OPTIONS)
+        list(APPEND arg_OPTIONS ${VCPKG_MAKE_CONFIGURE_OPTIONS})
+    endif()
+    if(DEFINED VCPKG_MAKE_CONFIGURE_OPTIONS_RELEASE)
+        list(APPEND arg_OPTIONS_RELEASE ${VCPKG_MAKE_CONFIGURE_OPTIONS_RELEASE})
+    endif()
+    if(DEFINED VCPKG_MAKE_CONFIGURE_OPTIONS_DEBUG)
+        list(APPEND arg_OPTIONS_DEBUG ${VCPKG_MAKE_CONFIGURE_OPTIONS_DEBUG})
+    endif()
+
+    set(src_dir "${arg_SOURCE_PATH}")
+
+    z_vcpkg_warn_path_with_spaces()
+
+    set(prepare_flags_opts "")
+    if(arg_DISABLE_MSVC_WRAPPERS)
+        list(APPEND prepare_flags_opts "DISABLE_MSVC_WRAPPERS")
+    else()
+        
+    endif()
+    if(arg_DISABLE_CPPFLAGS)
+        list(APPEND prepare_flags_opts "DISABLE_CPPFLAGS")
+    endif()
+    if(DEFINED arg_LANGUAGES)
+        list(APPEND prepare_flags_opts "LANGUAGES" "${arg_LANGUAGES}")

```

```suggestion
        list(APPEND prepare_flags_opts "LANGUAGES" ${arg_LANGUAGES})
```

LANGUAGES is a multi-arg so it shouldn't be quoted.